packages/v4/src/content/design-guidelines/content/units-and-symbols.md

```diff
@@ -0,0 +1,102 @@
+---
+id: Units and symbols
+section: UX writing
+---
+
+This page focuses on how to write units and symbols in an interface. For guidance on writing and displaying numeric values like date, time, and currency, see our [Numerics](/ux-writing/numerics) resource.
```

Comment:
We can delete this intro! I think the nav item labels we have are informative enough for users, and this page's TOC lets users know what we'll cover 👍